apps/docs/src/app/documentation/core-helpers/code-example/code-example.component.ts

```diff
@@ -32,23 +35,227 @@ export class CodeExampleComponent implements OnInit, AfterViewInit {
     @Input()
     exampleFiles: ExampleFile[] = [];
 
+    // @Input() fileName: string;
```

Comment:
do we need those?

Reply:
yes we do there passed in each docs files